src/containers/Search.js

```diff
@@ -130,11 +136,16 @@ class search extends React.Component {
         const indexOfLastTodo = currentPage * todosPerPage;
         const indexOfFirstTodo = indexOfLastTodo - todosPerPage;
         if (loading)
-        return <div>Fetching</div>
-        if (error)
-        return <div>Error</div>
-
-        if( Object.keys(checkResult).length >0) {
+        return <Container><div class="d-flex justify-content-center"><FontAwesomeIcon icon={faSpinner} size="3x" spin/></div></Container>
+        if (error) 
+            if (error == "SearchError") {
+              return <Container><div class="d-flex justify-content-center">{(localizer.lang === 'en_CA') ? "Oups, the search encountered a temporary error and could not complete your request. Please try again or contact our helpdesk." : "Oups, la fonction de recherche à rencontrer une erreur temporaire et ne peut completer la requête. S'il vous plaît, veuillez réessayer plus tard ou contacter notre bureau d'aide." }</div></Container>      
```

Comment:
Just a small typo in the English messages. Oups should be Oops, but after that everything looks good!